chpl/chpl-resources/src/main/resources/email.properties

```diff
@@ -30,7 +30,7 @@ during Log In, or by contacting your local administrator.
 #password changed email
 user.updatedPassword.subject=CHPL Password Changed
 user.updatedPassword.heading=Your Password Has Been Changed
-user.updatedPassword.paragraph1=We successfully processed a request to change your�password on %s.
+user.updatedPassword.paragraph1=We successfully processed a request to change your password on %s.
```

Comment:
Something looks weird on this line, but I can't do a "suggested code update" to fix it... Do you see the diff here?

Reply:
Wow.. this line is highlighted as a diff for me, but nothing looks odd about it. In the email Github sent me with your comment, however, I saw a non-UTF-8 character.